pkgs/applications/graphics/sane/backends/brscan5/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
     tar xfv data.tar.xz
   '';
 
-  nativeBuildInputs = [ makeWrapper patchelf coreutils ];
+  nativeBuildInputs = [ makeWrapper patchelf ];
```

Comment:
Isn't patchelf also part of the stdenv?

Reply:
Only on linux stdenv, not darwin.

Reply:
Darwin isn't using elf objects. So you shouldn't require it there or am I missing something?